### DISCUSSION TITLE
Allow CMIs bperm check for ressidence permission

### DISCUSSION THREAD

**Discussion by CloudeLecaw** [Zrips/CMI#7648] [Enhancement]
Currently the bperm only work for cmi.
but specialy when creating custom aliases it would often be very very usefull if permissions could also be bypassed like it can be done
in cmi.

So could you add the option to accept the bperm of CMIs specialzed commands? :3

i aslo wrote a reuqest on Ressidence github because i dont know if this is a job of cmi to add or ressidence.
https://github.com/Zrips/Residence/issues/1008

**Comment by MrCrazys** [comment]
It's intentionally done like that. As far as I know, finding the base permission node is very complicated. Besides, CMI isn't designed to hook into other plugins really. If it were to be done for Residence, it would have to be done for all plugins. Seems totally unnecessary to me.

**Reply by CloudeLecaw** [reply]
why it should it unnessary to bypass other permissions then the cmi one?

Everyone is free to make suggestions here. and lucklie its not up to you to decide what is unnessary and what not ;0

should i request all your discussion request now with "its unnessary because of...." if i feel like its not needed for me?

in my eyes it was and is a usfull idea where i see many things and places where this could be used. and so i requested it. thats what suggestions are made for. it doesnt matter if its hard ti develop or not. noone said Zrips needs to accept all ideas from us. Else cmi would have a size of 23mb today ;)

**Comment by CloudeLecaw** [comment]
well may your right, but in this case because Ressidence is also made from Zrips too, i thought it might be a possible thing to add it for him.
